integrated_channels/integrated_channel/models.py

```diff
@@ -881,41 +881,58 @@ class Meta:
     resolved = models.BooleanField(default=False)
 
 
-class IntegratedChannelAPIRequestLogs(TimeStampedModel):
+class BaseIntegratedChannelAPIRequestLogs(TimeStampedModel):
     """
-     A model to track basic information about every API call we make from the integrated channels.
+    A model to track basic information about every API call we make from the integrated channels.
     """
+
     enterprise_customer = models.ForeignKey(
-        EnterpriseCustomer, on_delete=models.CASCADE)
+        EnterpriseCustomer, on_delete=models.CASCADE
+    )
     enterprise_customer_configuration_id = models.IntegerField(
-        blank=False, null=False, help_text='ID from the EnterpriseCustomerConfiguration model')
-    endpoint = models.TextField(blank=False, null=False)
+        blank=False,
+        null=False,
+        help_text="ID from the EnterpriseCustomerConfiguration model",
+    )
+    endpoint = models.TextField(
```

Comment:
Will this field be saving urls? If yes, we can use Django's URLField. It's just a CharField with url validation. https://docs.djangoproject.com/en/5.0/ref/models/fields/#urlfield